.coveragerc

```diff
@@ -757,9 +756,10 @@ omit =
     homeassistant/components/sensor/mvglive.py
     homeassistant/components/sensor/nederlandse_spoorwegen.py
     homeassistant/components/sensor/netatmo_public.py
-    homeassistant/components/sensor/netdata.py
     homeassistant/components/sensor/netdata_public.py
```

Comment:
There is no `netdata_public` sensor platform.

Reply:
I did not add this line, it was simply sorted in place since the rest was sorted alphabetically.